includes/class-wc-payments-token-service.php

```diff
@@ -60,17 +60,25 @@ public function add_token_to_user( $payment_method, $user ) {
 		// Clear cached payment methods.
 		$this->customer_service->clear_cached_payment_methods_for_user( $user->ID );
 
-		if ( Payment_Method::SEPA === $payment_method['type'] ) {
-			$token = new WC_Payment_Token_WCPay_SEPA();
-			$token->set_gateway_id( CC_Payment_Gateway::GATEWAY_ID );
-			$token->set_last4( $payment_method[ Payment_Method::SEPA ]['last4'] );
-		} else {
-			$token = new WC_Payment_Token_CC();
-			$token->set_gateway_id( CC_Payment_Gateway::GATEWAY_ID );
-			$token->set_expiry_month( $payment_method[ Payment_Method::CARD ]['exp_month'] );
-			$token->set_expiry_year( $payment_method[ Payment_Method::CARD ]['exp_year'] );
-			$token->set_card_type( strtolower( $payment_method[ Payment_Method::CARD ]['brand'] ) );
-			$token->set_last4( $payment_method[ Payment_Method::CARD ]['last4'] );
+		switch ( $payment_method['type'] ) {
+			case Payment_Method::SEPA:
+				$token = new WC_Payment_Token_WCPay_SEPA();
```

Comment:
I find it a bit odd, that token classes are not declared in the `use ...` section

Reply:
Those are in the same folder and namespace, hence it works without imports.